app/server/src/controllers/odinController.ts

```diff
@@ -1,22 +1,22 @@
 import { Application, Request, Response } from "express";
 import { exampleOdinModel } from "../exampleOdin/exampleOdinModel";
-import { exampleOdinUtils } from "../exampleOdin/exampleOdinUtils";
+import { exampleOdinRunner } from "../exampleOdin/exampleOdinRunner";
 
 export class OdinController {
     private readonly _path = "/odin";
 
     registerRoutes = (app: Application) => {
-        app.get(`${this._path}/utils`, OdinController.getUtils);
+        app.get(`${this._path}/runner`, OdinController.getRunner);
         app.get(`${this._path}/model`, OdinController.getModel);
     };
 
     private static addHeader = (res: Response) => {
         res.header("Content-Type", "application/javascript");
     };
```

Comment:
Good idea, but I think I'll put it in a controller base class rather than global. Should only be controllers which are interacting with the responses.